common/ext/class.GenerisInstaller.php

```diff
@@ -48,16 +40,21 @@ public function install()
         }
  
         $this->installLoadDefaultConfig();
-        
+
+        // Id of the writable model.
+        /** @var ModelFactory $modelFactory */
+        $modelFactory = $this->getServiceManager()->get(ModelFactory::SERVICE_ID);
+        $writableModelId = $modelFactory->getModelId(LOCAL_NAMESPACE);
+
         $model = new \core_kernel_persistence_smoothsql_SmoothModel(array(
             \core_kernel_persistence_smoothsql_SmoothModel::OPTION_PERSISTENCE => 'default',
-            \core_kernel_persistence_smoothsql_SmoothModel::OPTION_READABLE_MODELS => array('1'),
-            \core_kernel_persistence_smoothsql_SmoothModel::OPTION_WRITEABLE_MODELS => array('1'),
-            \core_kernel_persistence_smoothsql_SmoothModel::OPTION_NEW_TRIPLE_MODEL => '1',
+            \core_kernel_persistence_smoothsql_SmoothModel::OPTION_READABLE_MODELS => [$writableModelId],
```

Comment:
already part of https://github.com/oat-sa/generis/pull/686/files#diff-cb45d9e73f3ab29ea11eadbdc095c55fR59